gitea/repositories/mock-turtle/examples/09 - Eelectron-quick-start/package-lock.json

```diff
@@ -439,7 +439,12 @@
       "integrity": "sha512-j5EctnkH7amfV/q5Hgmoal1g2QHFJRraOtmx0JpIqkxhBhI/lJSl1nMpQ45hVarwNETOoWEimndZ4QK0RHxuxQ==",
       "dev": true
     },
-
+    "minimist": {
```

Comment:
:exclamation:**Cycode**: Security vulnerability found in newly introduced dependency.
Dependency: minimist
Ecosystem: NPM
Identifiers: CVE-2021-44906
Severity: Critical
Fixed in version: 1.2.6
Dependency Paths: electron 7.2.4 -> extract-zip 1.7.0 -> mkdirp 0.5.5 -> minimist 1.2.5
Direct Dependency: No
Development Dependency: Yes
Vulnerability Description: Minimist <=1.2.5 is vulnerable to Prototype Pollution via file index.js, function setKey() (lines 69-95).
 <!-- bef20612-2c6b-4402-b742-d48569a938bf -->